### PR TITLE
Creating parser script and new mapping for summary data

### PR DESCRIPTION
**Pull request by Alexey-ebi**
This indices will be used by new 'Summary' tab on data portal

### CODE REVIEW

File: elasticsearch/summary_specimen.mapping.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+          "type": "keyword"
+        },
+        "value": {
+          "type": "keyword"
+        }
+      }
+    },
+    "breedSummary": {
+      "properties": {
+        "name": {
```
Comment (wizardfan) [inline elasticsearch/summary_specimen.mapping.json]: same comment, rename this field to something like speciesName

File: scripts/create_summary.py [hunk 2]
```diff
@@ -18,7 +19,85 @@ def __init__(self, es_instance, logger_instance):
...
+            else:
+                paper_published_data['no'] += 1
+
+            # get data for organism_data
+            organism = item['_source']['organism']['text']
+            organism_data.setdefault(organism, 0)
+            organism_data[organism] += 1
+
+            # get data for breeds_data
+            organism = item['_source']['organism']['text']
```
[the remark this answers is not in the document]
Comment (Alexey-ebi) [reply scripts/create_summary.py]: Agree, deleted.

File: elasticsearch/summary_specimen.mapping.json (hunk 1 above)
Comment (Alexey-ebi) [reply elasticsearch/summary_specimen.mapping.json]: agree, done

File: elasticsearch/summary_organism.mapping.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      }
+    },
+    "breedSummary": {
+      "properties": {
+        "name": {
+          "type": "keyword"
+        },
+        "value": {
+          "properties": {
+            "name": {
```
[the remark this answers is not in the document]
Comment (Alexey-ebi) [reply elasticsearch/summary_organism.mapping.json]: agree, done

File: scripts/create_summary.py [hunk 4]
```diff
@@ -18,7 +19,85 @@ def __init__(self, es_instance, logger_instance):
...
+                tmp_list.append({
+                    'name': tmp_k,
+                    'value': tmp_v
+                })
+            results['breedSummary'].append({
+                "name": k,
+                "value": tmp_list
+            })
+        body = {"doc": results}
+        self.es_instance.update(index="summary_organism", doc_type="_doc", id="summary_organism", body=body)
```
[the remark this answers is not in the document]
Comment (Alexey-ebi) [reply scripts/create_summary.py]: Added index function instead of update and it will add or update document (if it already exists)

**Pull request merged**